boards/opentitan/src/main.rs

```diff
@@ -286,20 +291,22 @@ pub unsafe fn reset_handler() {
         ]
     );
 
-    let cdc = components::cdc::CdcAcmComponent::new(
+    let ctap_send_buffer = static_init!([u8; 64], [0; 64]);
+    let ctap_recv_buffer = static_init!([u8; 64], [0; 64]);
+
+    let (ctap, ctap_driver) = components::ctap::CtapComponent::new(
         &earlgrey::usbdev::USB,
-        capsules::usb::cdc::MAX_CTRL_PACKET_SIZE_NRF52840,
```

Comment:
It seems to me that the max packet size should be provided to the API for CTAP as well. But maybe we should rather fix #2123 first instead?

Reply:
CTAP requires a max packet of 64 bytes, I don't think it makes sense to specify.

Reply:
Then it will break for the current sam4l, right?

Reply:
Yes, the SAM4L hardware can not meet the CTAP spec of a max packet size of 64 bytes, https://fidoalliance.org/specs/fido-v2.0-id-20180227/fido-client-to-authenticator-protocol-v2.0-id-20180227.html#usb-descriptors